validator/webextension/_locales/en/messages.json

```diff
@@ -48,5 +48,14 @@
         "content": "$1"
       }
     }
+  },
+  "warningCanonicalDoesntLinkToAMP": {
```

Comment:
nit: change Doesnt to DoesNot

Reply:
Addressed with https://github.com/ampproject/amphtml/pull/11809/commits/a4656bf26f90d75e40ef02102d2bdbd8ce18a4b8.